docs/patching/fixture-personality-options.md

```diff
@@ -186,18 +191,20 @@ Sometimes you may want to modify the way a fixture behaves, or you may
 find a fixture personality contains errors and needs editing. You can
 edit personalities directly on the console.
 
-1. Press \<Patch\>, \[Edit Fixtures\], .
+1. Press \<Patch\>.
+
+2. Press \[Edit Fixtures\].
 
-2. Press \[Edit Personality\].
+3. Press \[Edit Personality\].
 
-3. The softkeys show a list of all fixture types patched in your show.
+4. The softkeys show a list of all fixture types patched in your show.
 Select the fixture type to be edited.
 
-4. Personality Builder will open allowing you to edit the personality.
+5. Personality Builder will open allowing you to edit the personality.
 
-5. When you save changes, your show will be updated with the edited
-personality. Your changes are also saved to the fixture library on the
-console.
+6. When you save changes, your changes are saved as user personality 
```

Comment:
```suggestion
6. When you save changes, your changes are saved to the user personality 
```

Reply:
No - it really is like this: if you edit a factory personality and attempt to save it it automatically saves as a (new) user personality.